skimage/future/manual_segmentation.py

```diff
@@ -21,6 +19,8 @@ def _mask_from_vertices(vertices, shape, label):
 
 
 def _draw_polygon(ax, vertices, alpha=0.4):
+    import matplotlib.pyplot as plt
```

Comment:
This one doesn't seem necessary.

Reply:
@soupault there is a plt.draw() at line 24... Tricked by the code folding? :)

Reply:
tricky cold folding.

Reply:
>Tricked by the code folding? :)

Nope :). `_draw_polygon` is called only from `manual_polygon_segmentation` and `manual_lasso_segmentation`, which already contain the required import in their scope.

Reply:
line 27?

Reply:
Hmm... Ignore my comments. Apparently, works only with closures.